go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/hashicorp/terraform-plugin-framework v1.13.0
 	github.com/hashicorp/terraform-plugin-framework-validators v0.15.0
 	github.com/hashicorp/terraform-plugin-go v0.25.0
+	github.com/hashicorp/terraform-plugin-log v0.9.0
```

Comment:
Take latest commit for gopowerstore

Reply:
Need to revert the changes done on FSModify Type, will take the latest commit after merge.

Reply:
Updated the gopowerstore version to the latest.